includes/core/MySQLiHelpers.php

```diff
@@ -17,6 +17,7 @@ public function __construct(\mysqli $conn) {
 	 * @return mixed an associative array of results for selects, affected rows for updates, insert_id for inserts and null on any error
 	 */
 	public function query(string $query, string $types = null, array $params = null, bool $single = false) {
+		$query = trim($query);
```

Comment:
:+1:

Reply:
We're using all this new stuff at Docket. :) Thanks again, Ryan. <3